### PR TITLE
[ICDS] AWW report year/month dropbox restrictions added

### PR DESCRIPTION
**Pull request by dborowiecki**
Hi @calellowitz @Rohit25negi

I updated year/month dropbox options in AWW report, TICKET: https://dimagi-dev.atlassian.net/browse/ICDS-675

Need QA: no
Environment: ICDS
locally tested: yes

### CODE REVIEW

File: custom/icds_reports/static/js/directives/download/download.directive.js [hunk 1]
```diff
@@ -342,9 +344,34 @@ function DownloadController($rootScope, $location, locationHierarchy, locationsS
...
+
+                if (vm.selectedMonth > vm.months.slice(-1)[0].id) {
+                    vm.selectedMonth = vm.months.slice(-1)[0].id;
+                }
+            } else {
+                if (vm.selectedYear === 2018) {
+                    vm.months = vm.months.slice(-3);
+
+                    if (vm.selectedMonth < 10) {
+                        vm.selectedMonth = 12;
```
Comment (Rohit25negi) [inline custom/icds_reports/static/js/directives/download/download.directive.js]: Should not it be set to october. as the ticket says that 'We want the filter options to be available from October 2018'

Comment (Rohit25negi) [reply custom/icds_reports/static/js/directives/download/download.directive.js]: It currently looks that it sets to December.

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
Left One Comment. @dhiraj-beehyv  It looks you will be handling this PR Since Lukaz has left

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
@dhiraj-beehyv  can you also test this locally. The expected behaviour from the ticket is that the report should be available from oct 2018. and the year and month filter should behave accordingly

**Comment by dborowiecki** [comment]
Changes implemented in [this PR](https://github.com/dimagi/commcare-hq/pull/25250). Closing.

**Pull request closed without merging**